valtimo-dependencies/build.gradle

```diff
@@ -53,18 +53,26 @@ dependencies {
         api("com.ritense.valtimo:form-link:${projectVersion}!!")
         api("com.ritense.valtimo:form:${projectVersion}!!")
         api("com.ritense.valtimo:keycloak-iam:${projectVersion}!!")
+        api("com.ritense.valtimo:local-document-generation:${projectVersion}!!")
```

Comment:
Not sure if `local-document-generation` should be included. Because if it isn't included, the document-generation will fail-fast

Reply:
Same. It's just a version constraint.